clean_project/construct_dict.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""
+Construct a JSON file used to indexing all words in a config_file.yaml
+"""
+import argparse
+import codecs
+import json
+import os
+import yaml
```

Comment:
ok it is an external library then. we need to find a way to bundle the application with this library too then, we don't want the client to install several random libraries on his own